animareflection/ui
# Flyout Component

## Description

##### Task link: https://trello.com/c/WGrVFpNm/117-flyout-component

Added `Flyout` component.

## Test Steps

1. Ensure the component's structure, stories, and testing suite are robust.
2. Confirm that all tests are successful.
3. Check that the `Flyout` displays correctly in both storybook and sample downstream applications.
